src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CatalogPriceRuleController.php

```diff
@@ -75,6 +79,15 @@ public function indexAction(
         ]);
     }
 
+    public function productFormListAction(): JsonResponse
```

Comment:
You need to add the security annotation You always forget them but this is mandatory on ALL the controllers

Reply:
You're right I think in this case you can add a "double" permission and you'll need to write it manually: `     * @AdminSecurity("is_granted('read', 'AdminProducts') || is_granted('read', 'AdminSpecificPriceRule')")
`